src/Services/VideoService.php

```
@@ -71,13 +57,6 @@ public function relatedToVideoId($part, $params) {
 
 			$service = new \Google_Service_YouTube($this->client);
 			return $service->search->listSearch($part, $params);
-
-		} catch (\Google_Service_Exception $e) {
-			throw new Exception($e->getMessage(), 1);
-
-		} catch (\Google_Exception $e) {
-			throw new Exception($e->getMessage(), 1);
-
 		} catch (Exception $e) {
 			throw new Exception($e->getMessage(), 1);
 		}
```

why u meage try {} catch {} if u catch all exceptions and throw Exception ?